web/src/pages/EditS3LogSource/EditS3LogSource.tsx

```diff
@@ -67,6 +67,7 @@ const EditS3LogSource: React.FC = () => {
         { prefix: '', logTypes: [] },
       ],
       kmsKey: data?.getS3LogIntegration?.kmsKey ?? '',
+      managedBucketNotifications: data?.getS3LogIntegration?.managedBucketNotifications,
```

Comment:
Default value should be  `false` I presume

```suggestion
      managedBucketNotifications: data?.getS3LogIntegration?.managedBucketNotifications ?? false,
```

Reply:
I added just to be consistent, but the default value has no meaning since we don't display it anywhere.

And if we dont get the actual value, then the flow will break in more than one place